data/projects.json

```diff
@@ -108,5 +108,16 @@
         "level": "basic",
         "site": "",
         "tags": "feature-maps-and-filter-visualizer, machine-learning, python"
+    },
+    {
+        "name": "TravelYaari A traveling website",
+        "display": "TravelYaari",
+        "description": "This project is based on how an effective travel website will make us feel. It is based on a traveling website with fully-featured functions that will activate the traveling bug with vibrant imagery. This website contains highlights of some important places along with high-quality photography and allows people to book their dream destination within their budgets. It also includes full customer support, an easy payment system, book as per your time choice, full virtual tour of place through different videos and images.",
+        "author": "shsarv",
+        "category": "website",
+        "level": "intermediate",
+        "site": "https://travelyaari.herokuapp.com/",
```

Comment:
The URL doesn't work
![image](https://user-images.githubusercontent.com/37890026/119829612-6a890280-bec9-11eb-9320-6b3d6d6919b2.png)

Reply:
[https://travelyaari.herokuapp.com/](https://travelyaari.herokuapp.com/)  @ming-tsai  , the URL is working fine. it may little longer time for the first time due to heavy data. Please rerun the URL because its completely working on my system.
![resove](https://user-images.githubusercontent.com/55739302/119869949-75ca4580-bf3e-11eb-889b-6afced6ddcac.png)